CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.16)
 # Set current version of the project
 set(TARAXA_MAJOR_VERSION 1)
 set(TARAXA_MINOR_VERSION 9)
-set(TARAXA_PATCH_VERSION 1)
+set(TARAXA_PATCH_VERSION 2)
```

Comment:
btw we dont really check patch version anywhere, we only send it in status packet and log it... is that intended behaviour ?

Reply:
yes, it was requirement from Justin not to check patch version anywhere (as it should not be breaking change)

Reply:
@JakubFornadel  well these version checks are only for our own convenience during testnet stage.  During mainnet we will only be able to defend checking protocol version, because of course anyone can implement their own node design.    So I suggested that we only check major and minor version and leave patch free to be increased without requiring other nodes to upgrade to interoperate, for example for performance and other chores.